src/main/java/com/artipie/asto/Storage.java

```diff
@@ -62,8 +62,19 @@
      *
      * @param key The key of value.
      * @return Size of value in bytes.
+     * @deprecated Use {@link #metadata(Key)} to get content size
      */
-    CompletableFuture<Long> size(Key key);
+    @Deprecated
```

Comment:
hm, with this default implementation, what is the real reason to deprecate this method? We will repeat the same code in the adapters...

Reply:
The reason for making it `default` is to keep backward compatibility, we'll be able to update this dependency in `artipie/artipie` and keeping all adapters working. The reason of deprecation is to require all adapters eventually replace usage of this method with new one due to compilation warning. So later we'll remove it safely.

Reply:
Sorry, seems like I do not express my point clear enough. I do not think, that this default implementation should be deprecated, because to obtain the size of the storage item in the adapters we will repeat the same code, that is written in this default implementation.

Reply:
I think we have to avoid duplicating functionality in one place, without deprecation it would be possible to use two different methods of `Storage` to get the size. If using `meta` method would be not comfortable, we can introduce additional helper object with frequently used operations, e.g. `long size = new MetaAttributes(meta).size()`, WDYT?

Reply:
ok, sounds good